README.wcf.md

```diff
@@ -78,9 +78,9 @@ Tag | Dockerfile
 Tag | Dockerfile
 ---------| ---------------
 4.8-20200211-windowsservercore-ltsc2016, 4.8-windowsservercore-ltsc2016, 4.8, latest | [Dockerfile](https://github.com/microsoft/dotnet-framework-docker/blob/master/4.8/wcf/windowsservercore-ltsc2016/Dockerfile)
-4.7.2-20200211-windowsservercore-ltsc2016, 4.7.2-windowsservercore-ltsc2016, 4.7.2 | [Dockerfile](https://github.com/microsoft/dotnet-framework-docker/blob/master/4.7.2/wcf/windowsservercore-ltsc2016/Dockerfile)
-4.7.1-20200211-windowsservercore-ltsc2016, 4.7.1-windowsservercore-ltsc2016, 4.7.1 | [Dockerfile](https://github.com/microsoft/dotnet-framework-docker/blob/master/4.7.1/wcf/windowsservercore-ltsc2016/Dockerfile)
-4.7-20200211-windowsservercore-ltsc2016, 4.7-windowsservercore-ltsc2016, 4.7 | [Dockerfile](https://github.com/microsoft/dotnet-framework-docker/blob/master/4.7/wcf/windowsservercore-ltsc2016/Dockerfile)
+4.7.2-20200310-windowsservercore-ltsc2016, 4.7.2-windowsservercore-ltsc2016, 4.7.2 | [Dockerfile](https://github.com/microsoft/dotnet-framework-docker/blob/master/4.7.2/wcf/windowsservercore-ltsc2016/Dockerfile)
```

Comment:
I think the timestamp should be updated to today.  This could be a source of confusion.